frontend/Dockerfile

```diff
@@ -5,10 +5,12 @@ RUN mkdir /app
 WORKDIR /app
 
 # Install app dependencies
-COPY . /app/
+COPY package.json package-lock.json ./
 
 RUN npm ci
 
+COPY . .
```

Comment:
I would expect this to overwrite the node_modules, which were just created with `npm ci`. Is that not the case?

I think everyone will have a node_modules folder on the host because of the developer tooling (eslint, prettier, etc).

Reply:
Yeah, `COPY` should respect the .dockerignore file - I haven't run into any issues with it recently. I think there may be an issue *if* the Dockerfile isn't in the same directory as the .dockerignore? 

This should substantially speed up building the docker images locally though. Might be good to compare.